workloads/router-perf-v2/http-perf.yml

```diff
@@ -10,17 +10,17 @@ jobs:
   waitWhenFinished: true
   objects:
     - objectTemplate: templates/nginx-pod.yml
-      replicas: ${NUMBER_OF_ROUTES}
+      replicas: {{ .NUMBER_OF_ROUTES }}
```

Comment:
`http-perf.yml` is now duplicating `http-perf-depl.yml`
The difference before was that `http-perf-depl.yml` was using deployments, and `http-perf.yml` was deploying pods. But you just changed the contents of the `templates/nginx-pod.yml` to be a deployment despite the filename and `templates/nginx-deploy.yml` already existing?